src/main/java/org/apache/commons/io/IOUtils.java

```diff
@@ -814,9 +814,21 @@ public static boolean contentEquals(final Reader input1, final Reader input2)
     }
 
     private enum LastState {
-        r,
-        normal,
-        newLine;
+        /**
+         * If last char is '\r'.
+         */
+        R,
+
+        /**
+         * If last char is not '\r' nor '\n'.
+         */
+        NORMAL,
+
+        /**
+         * If we just moved to a new line.
+         * It might sounds weird but after you see the codes you can know it.
+         */
+        NEW_LINE;
```

Comment:
This could be NL to agree with CR above

Reply:
@sebbASF I changed the enum to int, and get even better performance result, so I deleted that enum.